mmv1/third_party/terraform/services/dataflow/resource_dataflow_flex_template_job_test.go.erb

```diff
@@ -299,8 +299,9 @@ func TestAccDataflowFlexTemplateJob_withKmsKey(t *testing.T) {
 
 	randStr := acctest.RandString(t, 10)
 	job := "tf-test-dataflow-job-" + randStr
-	key_ring := "tf-test-dataflow-kms-ring-" + randStr
-	crypto_key := "tf-test-dataflow-kms-key-" + randStr
+	kms := acctest.BootstrapKMSKeyInLocation(t, "us-central1")
+	key_ring := kms.KeyRing.Name
+	crypto_key := kms.CryptoKey.Name
```

Comment:
It looks like this should be the full name based on https://pkg.go.dev/google.golang.org/api/cloudkms/v1#CryptoKey